gadget/gadget_test.go

```diff
@@ -2275,7 +2275,7 @@ volumes:
 	}{
 		{mockOtherYaml, `cannot find entry for volume "volumename" in updated gadget info`},
 		{mockManyYaml, "gadgets with multiple volumes are unsupported"},
-		{mockBadStructureSizeYaml, `cannot lay out the new volume: cannot lay out volume, structure #0 \("bad-size"\) size is not a multiple of sector size 512`},
+		{mockBadStructureSizeYaml, `incompatible layout change: incompatible change in the number of structures from 0 to 1`},
```

Comment:
shouldn't the mock yaml name be changed then?

Reply:
renamed